app/services/search/postgres/feed.rb

```diff
@@ -0,0 +1,103 @@
+module Search
+  module Postgres
+    class Feed
+      FIELDS_TO_EXCLUDE_FROM_SERIALIZATION = %i[
+        approved body_text experience_level_rating
+        experience_level_rating_distribution featured featured_number
+        hotness_score language published reactions_count score
+      ].freeze
+
+      def self.search_documents(
+        term: nil, approved: nil, class_name: nil, id: nil, sort_by: nil, sort_direction: nil, published_at: nil,
+        page: 1, per_page: 60
+      )
+        page = (page || 1).to_i + 1 # NOTE: the frontend starts at 0, need to fix this
```

Comment:
Postgres treats page 0 as if it were page 1, so we can't really just add `+1` to it, but we can always change the frontend start page when we're done and remove this bug